frontend/src/pages/Player/MetadataBox/MetadataBox.tsx

```diff
@@ -171,145 +284,12 @@ export const MetadataBox = React.memo(() => {
 				paddingBottom="6"
 				gap="4"
 			>
-				<TableList
-					truncateable
-					data={[
-						{
-							keyDisplayValue: 'Email',
-							valueDisplayValue: displayValue,
-						},
-						{
-							keyDisplayValue: 'UserID',
-							valueDisplayValue: session?.fingerprint?.toString(),
-						},
-						{
-							keyDisplayValue: 'Location',
-							valueDisplayValue: geoData,
-						},
-						{
-							keyDisplayValue: 'Browser',
-							valueDisplayValue:
-								session?.browser_name &&
-								session?.browser_version
-									? `${
-											session.browser_name
-									  } ${getMajorVersion(
-											session.browser_version,
-									  )}`
-									: undefined,
-						},
-						{
-							keyDisplayValue: 'OS',
-							valueDisplayValue:
-								session?.os_name && session?.os_version
-									? `${session.os_name} ${getMajorVersion(
-											session.os_version,
-									  )}`
-									: undefined,
-						},
-						{
-							keyDisplayValue: 'Time',
-							valueDisplayValue: created.toLocaleString('en-us', {
-								hour: '2-digit',
-								minute: '2-digit',
-								timeZoneName: 'short',
-								day: 'numeric',
-								month: 'short',
-								weekday: 'long',
-								year:
-									created.getFullYear() !==
-									new Date().getFullYear()
-										? 'numeric'
-										: undefined,
-							}),
-						},
-					]}
-				/>
+				<TableList truncateable loading={loading} data={userData} />
 			</Box>
 		</Box>
 	)
 })
 
-export const UserDetailsBox = ({
```

Comment:
Fixes https://linear.app/highlight/issue/HIG-3627/enhanced-details-should-be-rendered-within-show-more